modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-namespace-scoped-argo-rollouts-installation_{context}"]
+= Configuring a namespace-scoped Argo Rollouts installation
+
+To install a namespace-scoped instance of Argo Rollouts, complete the following steps:
+
+.Prerequisites
+
+* You have logged in to the {product-title} cluster as an administrator.
```

Comment:
```suggestion
* You are logged in to the {product-title} cluster as an administrator.
```
I think it's reasonable to use present tense. You want to the reader to definitely be logged in as they start this. Just a suggestion.

Reply:
Fixed.